mount_config.go

```diff
@@ -205,6 +205,11 @@ type MountConfig struct {
 	// without O_TRUNC, followed by a SetInodeAttributes op with the target size set to 0.
 	// Ref: https://github.com/torvalds/linux/commit/6ff958edbf39c014eb06b65ad25b736be08c4e63
 	EnableAtomicTrunc bool
+
+	// Disable ReadDirPlus capability so ReadDir is used instead. Simple
+	// directory queries (i.e. 'ls' without '-l') can be faster with
+	// ReadDir
+	DisableReaddirplus bool
```

Comment:
I feel, EnableReaddirplus seems better option, as plan is to keep it by default off. Accordingly, we need to change the do, explaining the benefit of this feature.